CHANGELOG.txt

```diff
@@ -245,6 +245,7 @@
                     - Object.entries()
                     - Object.getPrototypeOf()
                 * Drop creme.widget.template(); Use String.prototype.template instead.
+                * Drop creme.forms.toImportField(); Use creme.widget.ImportField ('ui-import-field') instead.
```

Comment:
It was not deprecated, it's a direct breaking (.i.e you're in the bad section).

Reply:
My bad, I've done it for the other PR, not this one :sweat: